ledger/ledger_test.go

```diff
@@ -832,18 +831,13 @@ int 1
 	}}
 	a.NoError(l.appendUnvalidatedTx(t, initAccounts, initSecrets, appcall, ad))
 
-	rnd = l.Latest()
-	acctworRes, err := l.LookupApplication(rnd, creator, appIdx)
+	acctworRes, _, err := l.LookupApplication(creator, appIdx)
```

Comment:
In  many of such places in tests, where the returned round number is ignored, shouldn't it be tested? 
For example here, it returns a round number 4. Will be nice to verify it, so that in case something changes and brakes the behavior, the test can detect it....